tests/run.sh

```diff
@@ -17,10 +17,14 @@ if [ "$CI" = true ]; then
     set -x
 fi
 
-source_dir="$PWD"
+source_dir="$(dirname "$(realpath "$(type -p $0)")")"/..
```

Comment:
FWIW: I tend to use this snippet:

```sh
# Change to current dir (POSIXly).
unset CDPATH
cd -P -- "$(dirname -- "$0")"
```

In this case this would include a `/..`.